scripts/exrtma3d_post.ksh

```diff
@@ -207,18 +207,27 @@ if [ ! -s "${workdir}/wrfsubhnat.grib2" ]; then
 fi
 
 # transfer the output grib2 files to $COMOUTpost_rtma3d
-# add gust and howv (wave height) to the prslev and natlev files
+# add gust,vis, and howv (wave height) to the prslev and natlev files
 # change name from surface gust to 10-m gust
 
 # Note: NET should be RUN - AMG
-${WGRIB2} -V ${COMOUT}/${NET}.t${cyc}z.anl.gust.grib2 -set_lev "10 m above ground" -grib tmpgust.grib2
+# Change derived visibility (VIS:surface) to 2m visibility (VIS:2 m above ground)
+wgrib2 ${workdir}/wrfsubhprs.grib2 -match "VIS" -if "VIS:surface" -set_lev "2 m above ground" -grib ${workdir}/vis2mprs.grib2 -fi
+wgrib2 ${workdir}/wrfsubhnat.grib2 -match "VIS" -if "VIS:surface" -set_lev "2 m above ground" -grib ${workdir}/vis2mnat.grib2 -fi
+# Change analyzed gust from GUST:surface to GUST:10 m above ground
+${WGRIB2} -V ${COMOUT}/${RUN}.t${cyc}z.anl.gust.grib2 -set_lev "10 m above ground" -grib tmpgust.grib2
+# Remove surface visibility from grib2 files
+wgrib2 ${workdir}/wrfsubhprs.grib2 -not_if "VIS:surface" -grib ${workdir}/wrfsubhprs.grib2_no_sfcvis 
+wgrib2 ${workdir}/wrfsubhnat.grib2 -not_if "VIS:surface" -grib ${workdir}/wrfsubhnat.grib2_no_sfcvis
+# Overwrite wrfsubhprs and wrfsubhnat with renamed derived vis, analyzed vis, analyzed gust and analyzed wave height
 if [ "${RUN}" == "rtma3d" ]; then
```

Comment:
@AnnetteGibbs-NOAA Is this backwards? You have the "howv" data going into rtma3d files, but wouldn't that only be analyzed in urma3d?

Reply:
Yes, that is correct and will be removed from RTMA, but until we set up an URMA package, I think we need to keep this in.  We want to make sure that all aspects of the wave height works with all our future changes.  Then once we set up an URMA package, I will remove this from RTMA.

Reply:
OK, that makes sense. I'll go ahead and approve the PR!